
<launch>

<node pkg="mirracle_gestures" name="main" type="main.py" output="screen"/>

</launch>
